force_torque_sensor_broadcaster/test/test_force_torque_sensor_broadcaster.cpp

```diff
@@ -315,3 +292,12 @@ TEST_F(ForceTorqueSensorBroadcasterTest, All_InterfaceNames_Publish_Success)
   ASSERT_EQ(wrench_msg.wrench.torque.y, sensor_values_[4]);
   ASSERT_EQ(wrench_msg.wrench.torque.z, sensor_values_[5]);
 }
+
+int main(int argc, char ** argv)
+{
+  ::testing::InitGoogleTest(&argc, argv);
```

Comment:
we use Google Mock in all:

```suggestion
  ::testing::InitGoogleMock(&argc, argv);
```